Sources/PointFree/Admin/Index.swift

```diff
@@ -62,18 +64,15 @@ private let newEpisodeEmailRowView = View<Episode> { ep in
     ])
 }
 
-let sendNewEpisodeEmailMiddleware =
-  filterMap(require1 >>> pure, or: loginAndRedirect)
-    <<< requireAdmin
-    <<< filterMap(fetchEpisode >>> pure, or: redirect(to: .admin(.newEpisodeEmail(.show))))
-    <| { conn in pure(conn.map(get2)) }
-    >-> sendNewEpisodeEmails
+let sendNewEpisodeEmailMiddleware:
```

Comment:
This was taking 20 seconds to compile for me :(

I cleaned up some of the filtering logic and extracted some other stuff and now it's under 500 ms.